code/src/components/Counter.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+import { useSelector, useDispatch } from 'react-redux'
+import styled from 'styled-components'
+
+import todos from '../reducers/todos'
+
+const Counter = () => {
+  const allToDos = useSelector((store) => store.todos.items)
+  
+  const dispatch = useDispatch()
+  
+  const onCheckAll = () => {
+    dispatch(todos.actions.markAllTodos())
+  }
+  
+  const completedTodos = allToDos.filter((todo) => todo.isComplete)
+
+  return (
+    <Container>
+      <CompletedTodos>
+        {completedTodos.length} / {allToDos.length}
```

Comment:
Not sure what is considered best practice, but you could add the .length in const allTodos and const completedTodos instead to keep the code inside the return cleaner.